docs/index.md

```diff
@@ -29,7 +29,7 @@ Configure your project for use with UI5 Tooling.
 ❯ ui5 init
 Wrote ui5.yaml:
 
-specVersion: '2.6'
+specVersion: '3.0'
```

Comment:
should we use here also double quotes?

Reply:
IIRC that's how `ui5 init` does it because of the library we use for serializing to YAML 🤷‍♂️

Reply:
Ack